module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -371,7 +398,7 @@ namespace sys
     bool SystemManagerCommon::DestroyApplication(const std::string &name, Service *caller)
     {
         cpp_freertos::LockGuard lck(appDestroyMutex);
-        if (RequestServiceClose(name, caller)) {
+        if (RequestServiceClose(name, caller, 1000)) {
```

Comment:
What's 1000? Why is it different for apps than other services?

Reply:
temporary change, reverted, tx